docs/UserGuide.md

```diff
@@ -179,6 +179,72 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+### Creating a Tag: `addtag`
+
+Creates a new tag to Contax.
+
+Format: `addtag n/TAGNAME`
+
+* The `TAGNAME` parameter **must** be specified.
+* If the `TAGNAME` already exists (case-insensitive), the tag will not be created.
+
+Examples:
+* `addtag n/Potential Clients` Creates a tag named *Potential Clients* in the address book (if it does not exist).
+
+### Editing a Tag: `edittag`
+
+Edits an existing tag in Contax.
+
+Format: `edittag f/OLD_TAGNAME t/NEW_TAGNAME`
+
+* All parameters **must** be specified.
+* Edits a tag specified at `OLD_TAGNAME` and updates to the tag specified at `NEW_TAGNAME`.
+* `OLD_TAGNAME` and `NEW_TAGNAME` are case-insensitive.
+* If `OLD_TAGNAME` cannot be found, the command will not execute.
+* If `NEW_TAGNAME` already exists in Contax (case-insensitive), the command will not execute.
+
+Examples:
+* `edittag f/New Clients t/Prospective Clients` Changes the name of the tag *New Clients* to *Prospective Clients*. Command will be ignored if the tag *New Clients* does not exist in the first place or *Prospective Clients* already exists in Contax.
+
+### Listing All Tags: `listtags`
+
+Shows a list of all tags in Contax.
+
+Format: `listtags`
+
+**Example output**
+
+![List Tags](images/ListTag.png)
+
+### Deleting a Tag: `deletetag`
+
+Deletes the specified tag in Contax.
+
+Format: `deletetag n/TAGNAME`
+
+* The `TAGNAME` parameter **must** be specified.
+* Deletes the tag that matches the tag name (case-insensitive) e.g. *students* is the same as *Students*.
+* When the tag is deleted, all persons that contain this tag will have the tag removed.
+* If `TAGNAME` cannot be found, the command will not execute.
+
+Examples:
+* `deletetag n/friends` Deletes the tag named *friends* and disassociates any contacts that have the *friends* tag.
+
+### Finding Contacts by Tag: `findtag`
```

Comment:
I suggest naming it `findByTag` to make it more intutitive.

Reply:
I agree and have made the necessary changes.